DITAfication of Provisioning hosts (#4515)
theforeman/foreman-documentation: guides/common/modules/ref_troubleshooting-dns-problems.adoc

:_newdoc-version: 2.18.5

> :_template-generated: 2025-12-02

Reviewer: ```suggestion
```

:_mod-docs-content-type: REFERENCE

[id="troubleshooting-dns-problems_{context}"]
= Troubleshooting DNS problems

[role="_abstract"]

> DNS problems might prevent hosts from being provisioned because they cannot correctly resolve the {Project} or {SmartProxyServer} hostnames required to download installation content or configuration files.

Reviewer: ```suggestion
DNS problems might prevent hosts from being provisioned because they cannot correctly resolve {ProjectServer} or {SmartProxyServer} hostnames required to download installation content or configuration files.
```

> 

Author: New text, please review.

DNS server configuration considerations::
+
--
During the DNS record creation, {Project} performs conflict DNS lookups to verify that the host name is not in active use.
This check runs against one of the following DNS servers:

* The system-wide resolver if *Administer* > *Settings* > *Query local nameservers* is set to *true*.
* The nameservers that are defined in the subnet associated with the host.
* The authoritative NS-Records that are queried from the SOA from the domain name associated with the host.

If you experience timeouts during DNS conflict resolution, check the following settings:

* The subnet nameservers must be reachable from {ProjectServer}.
* The domain name must have a Start of Authority (SOA) record available from {ProjectServer}.
* The system resolver in the `/etc/resolv.conf` file must have a valid and working configuration.
--

> 

Reviewer: ```suggestion
```

trailing whitespace.
